doc/usage.rst

```diff
@@ -26,6 +26,7 @@ The ``--format`` option for the output format. Supported formats are ``txt`` (de
 NOTE: the output for the following formats are generated in accordance with schemas
 
 * ``checkstyle`` follows the common `"checkstyle" XML schema </doc/schemas/fix/checkstyle.xsd>`_
+* ``gitlab`` follows the `codeclimate JSON schema </doc/schemas/fix/codeclimate.json>`_
```

Comment:
this is a new format?
if so, why ticket is marked as bugfix?

Reply:
It's not new, it was missing from usage.rst because there was no schema defined

Reply:
got you!
can we put a test for this file content, that ensures the all reporters are present?

Reply:
It was done, see here ;) https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7089/files#diff-4b42eca6a4843666251d64385cf19ef941b679ff4fa63695ff2861d66bcc1846R163

Reply:
I meant not a test that gitlab-report output is matching the spec file, but that all possible report formats are enumareted in `doc/usage.rst` (like, you had to add new line - it should not be possible to forget adding the new report class to the docs and if one would, test should fail)

Reply:
i hoped for scout rule ;)

Reply:
I didn't even plan to add a schema file, but I decided since I'm already fixing Gitlab formatter I added it. Please don't punish me for that and add more and more on top of it 😅

Reply:
let's at least create a follow-up issue for that missing test

Reply:
Created #7106.